packages/gatsby/src/utils/develop.js

```diff
@@ -88,7 +88,7 @@ async function startServer(program) {
   app.use((req, res, next) => {
     const parsedPath = parsePath(req.originalUrl)
     if (parsedPath.extname === `` || parsedPath.extname === `.html`) {
-      res.sendFile(`${process.cwd()}/public/index.html`, err => {
+      res.sendFile(`${program.directory()}/public/index.html`, err => {
```

Comment:
Interesting that some of the CI deploys worked? Looks like we have may have false positives.